maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -43,25 +43,26 @@ class Maestro(
 
     private val sessionId = UUID.randomUUID()
 
-    private val cachedDeviceInfo by lazy {
-        fetchDeviceInfo()
-    }
-
-    private var screenRecordingInProgress = false
-
-    fun deviceName(): String {
-        return driver.name()
-    }
+    val deviceName: String
+        get() = driver.name()
+
+    private var _cachedDeviceInfo: DeviceInfo? = null
```

Comment:
aren't these two (this and the original code) functionally the same? My understanding is this is what `by lazy` does behind the scenes

Reply:
It is.

I changed it away from using `lazy` because I needed `_cachedDeviceInfo` to be a `var` instead of `val` for $reasons which are no longer true because I changed the code and forgot to bring it back.